src/form/Form.tsx

```diff
@@ -1336,7 +1327,7 @@ function Form({
           sendLoginCode({
```

Comment:
don't we also need a toggle to allow the user to optionally configure the ability to also submit the step and navigate to the next one in addition to sending the sms code or magic link?

Reply:
yes ty, I have that in the dashboard but missed it here - updated